src/lib/crypto/ecc/keystore.ts

```diff
@@ -0,0 +1,132 @@
+// import IDB from '../idb'
```

Comment:
We can recover files from git history, generally not great to include files that are entirely commented out but I also just did this with CI stuff so 🤷🏼‍♂️

Reply:
Sorry the whole ecc dir is not called becuase I did a quick switch into rsa. I can cealn that up

Reply:
Deleted that whole dir, refactored `rsa` to `tomb` -- we have a keystore meant to use with tomb, but just happens to use RSA